zulipterminal/model.py

```diff
@@ -576,10 +576,28 @@ def update_stream_message(
         response = self.client.update_message(request)
         display_error_if_present(response, self.controller)
         if response["result"] == "success":
-            old_topic = self.index["messages"][message_id].get("subject", None)
+            get_message = self.index["messages"]
+            old_topic = get_message[message_id].get("subject", None)
             new_topic = request["topic"]
+            stream_name = get_message[message_id].get("display_recipient", None)
+
+            recent_moved_msgs = ""
+            if propagate_mode == "change_all":
+                recent_moved_msgs = "all"
+            if propagate_mode == "change_later":
+                recent_moved_msgs = "some"
+            if propagate_mode == "change_one":
+                recent_moved_msgs = "one"
+
             if old_topic != new_topic:
-                self.controller.report_success("You changed a message's topic.")
+                if recent_moved_msgs == "one":
+                    self.controller.report_success(
+                        f"You changed {recent_moved_msgs} message's topic from #{stream_name} > {old_topic} to #{stream_name} > {new_topic}."
```

Comment:
This is identical except for the apostrophe placement? If so, it'll be simpler to integrate the difference into the variable, and avoid the need for a conditional here.

Reply:
This was what I referred to by simplifying the code in the next review.